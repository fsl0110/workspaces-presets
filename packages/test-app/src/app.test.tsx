import React from 'react';
import { shallow } from 'enzyme';
import App from './app';

describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
