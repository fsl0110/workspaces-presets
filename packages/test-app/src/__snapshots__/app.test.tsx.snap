// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<div>
  <h1
    data-testid="topic"
  >
    Hello React App !!
  </h1>
  <img
    src="test-file-stub"
  />
  <Main
    text="Hello high water"
  />
</div>
`;
